assets/queries/ansible/aws/http_port_open/query.rego

```diff
@@ -0,0 +1,113 @@
+package Cx
+
+CxPolicy [ result ] {
+    document := input.document[i]
+    tasks := getTasks(document)
+    task := tasks[t]
+    fromPort := task["amazon.aws.ec2_group"].rules[index].from_port
+    toPort := task["amazon.aws.ec2_group"].rules[index].to_port
+    cidr := task["amazon.aws.ec2_group"].rules[index].cidr_ip
+    cidr == "0.0.0.0/0"
+  	portNumber := 80
+  	fromPort <= portNumber
+  	toPort >= portNumber
```

Comment:
what if fromPort or toPort are set to -1? You should consider those edge cases as well